server/src/Packages/Paypal/Resources/js/templates/stripeFormTemplate.js

```diff
@@ -0,0 +1,23 @@
+const stripeFormTemplate = message => `
+<div id="stripeFormTemplate"  class="uk-container uk-container-xsmall uk-margin-large-top">
```

Comment:
@AlexeiaW I see there's reference to Stripe here, maybe not needed or just needs to be renamed?

Reply:
I removed this file